### PR TITLE
Hortense phase2

### PR DESCRIPTION
**Pull request by stdweird**

### CODE REVIEW

File: source/gent/tier1_hortense.rst [hunk 1]
```diff
@@ -617,3 +633,26 @@ Getting help
...
+
+In May 2023 a second phase was installed, adding 48 more nodes to the ``cpu_rome`` partition,
+20 extra GPU nodes with double the CPU and GPU memory in the new ``gpu_rome_a100_80`` partition and
+384 nodes using the newer AMD Milan CPUs called ``cpu_milan``.
+
+The Lustre based scratch storage was also doubled in volume and throughput to a total of 6PB.
+
+With the new GPU nodes, a renaming of the gpu node partitions occured. Users can mosyt likely still
+still use the same ``gpu_rome_a100`` partition that now includes all GPU nodes (and only select the
+``gpu_rome_a100_40`` or ``gpu_rome_a100_80`` for specific cases, e.g. when running multinode GPU jobs
```
Comment (stdweird) [inline source/gent/tier1_hortense.rst:649]: @boegel i only now realised that for multinode jobs, this will not be transparent

Comment (boegel) [reply source/gent/tier1_hortense.rst:649]: Yeah, for multi-node GPU jobs, they should probably be submitting to a more specific GPU partition.

Or would Slurm only start jobs within the same actual partition anyway?

File: source/gent/tier1_hortense.rst [hunk 2]
```diff
@@ -617,3 +633,26 @@ Getting help
...
+or explicitly requiring the larger amount of GPU/CPU memory of the ``gpu_rome_a100_80`` nodes).
+
+During the May 2023 maintenance, the OS and OFED infiniband stacks were updated to resp. RHEL 8.6
```
Comment (boegel) [inline source/gent/tier1_hortense.rst:652]: Maybe this should go into an "Recent updates" section below?

File: source/gent/tier1_hortense.rst (hunk 1 above)
Comment (itkovian) [reply source/gent/tier1_hortense.rst:649]: Yes, it will not pick nodes from separate partitions afaik.

File: source/gent/tier1_hortense.rst [hunk 3]
```diff
@@ -617,3 +633,26 @@ Getting help
...
+
+
+Phase 2
+-------
+
+In May 2023 a second phase was installed, adding 48 more nodes to the ``cpu_rome`` partition,
+20 extra GPU nodes with double the CPU and GPU memory in the new ``gpu_rome_a100_80`` partition and
+384 nodes using the newer AMD Milan CPUs called ``cpu_milan``.
+
+The Lustre based scratch storage was also doubled in volume and throughput to a total of 6PB.
```
Comment (itkovian) [inline source/gent/tier1_hortense.rst:645]:
```suggestion
The Lustre based scratch storage was also doubled in volume to a total of 6PB while increasing the overal throughput as well.
```

File: source/gent/tier1_hortense.rst [hunk 4]
```diff
@@ -12,19 +12,20 @@ General information
 **Hortense** is the 3rd VSC Tier-1 cluster, following *muk* (hosted by HPC-UGent, 2012-2016)
 and *BrENIAC* (hosted by HPC-Leuven, 2016-2022).
 
-The first phase of Hortense is also named ``dodrio``.
+The Hortense cluster also has an internal name ``dodrio``.
 
 It is available since 2021, is hosted by Ghent University,
 and maintained and supported by the HPC-UGent team.
+In 2023, a second phase was added more than doubling the capacity of the system.
```
Comment (itkovian) [inline source/gent/tier1_hortense.rst:19]:
```suggestion
In 2023 a second phase was added, more than doubling the existing capacity of the system.
```

File: source/gent/tier1_hortense.rst [hunk 5]
```diff
@@ -12,19 +12,20 @@ General information
...
 
 
 Hardware details
 ----------------
 
-Hortense currently consists of 3+1 partitions:
+Hortense currently consists of following partitions:
```
Comment (itkovian) [inline source/gent/tier1_hortense.rst:25]:
```suggestion
Hortense currently consists of the following partitions:
```

File: source/gent/tier1_hortense.rst [hunk 6]
```diff
@@ -12,19 +12,20 @@ General information
...
 
 - ``dodrio/cpu_rome``: main partition:
```
Comment (itkovian) [inline source/gent/tier1_hortense.rst:27]:
```suggestion
- ``dodrio/cpu_rome``: phase 1 main partition:
```

File: source/gent/tier1_hortense.rst [hunk 7]
```diff
@@ -33,20 +34,32 @@ Hortense currently consists of 3+1 partitions:
        - 2x 64-core AMD Epyc 7H12 CPU 2.6 GHz (128 cores per node)
        - 512 GiB RAM (~4GB/core), no swap
        - 480 GB SSD local disk
-- ``dodrio/gpu_rome_a100``: GPU partition:
+- ``dodrio/cpu_milan``: phase2 main partition:
```
Comment (itkovian) [inline source/gent/tier1_hortense.rst:37]:
```suggestion
- ``dodrio/cpu_milan``: phase 2 main partition:
```

File: source/gent/tier1_hortense.rst [hunk 8]
```diff
@@ -33,20 +34,32 @@ Hortense currently consists of 3+1 partitions:
...
+       - 2x 64-core AMD Epyc 7763 CPU 2.45 GHz (128 cores per node)
+       - 256 GiB RAM (~2GB/core), no swap
+       - 480 GB SSD local disk
+- ``dodrio/gpu_rome_a100_40``: GPU partition:
    - 20 workernodes, each with:
        - 2x 24-core AMD Epyc 7402 CPU 2.8 GHz (48 cores per node)
        - 4x NVIDIA A100-SXM4 (40 GB GPU memory), NVLink3
        - 256 GiB RAM (~5GB/CPU core), no swap
        - 480 GB SSD local disk
+- ``dodrio/gpu_rome_a100_80``: phase2 GPU partition:
```
Comment (itkovian) [inline source/gent/tier1_hortense.rst:48]:
```suggestion
- ``dodrio/gpu_rome_a100_80``: phase 2 GPU partition:
```

File: source/gent/tier1_hortense.rst [hunk 9]
```diff
@@ -512,7 +525,10 @@ performance benefit.
 To ensure that the paths which are 'engraved' in your own software installations always start with ``/readonly/``,
 for example in scripts or binaries that make part of the installation,
 you should install the software using the ``dodrio-bind-readonly`` utility. This allows you to "rename" the path to your
-project scratch directory so it starts with ``/readonly/``, while preserving write access to it.
+project scratch directory so it starts with ``/readonly/``, while preserving sort-of write access to it (
```
Comment (itkovian) [inline source/gent/tier1_hortense.rst:528]: what's up with the `(` at the end?

File: source/gent/tier1_hortense.rst (hunk 1 above)
Comment (stdweird) [reply source/gent/tier1_hortense.rst:649]: @itkovian can we set something in the partition definition that multinode jobs won't get started?

File: source/gent/tier1_hortense.rst (hunk 3 above)
Comment (stdweird) [reply source/gent/tier1_hortense.rst:645]: @itkovian the throughput was also doubled, i'll see if i can rephrase this better

Comment (stdweird) [reply source/gent/tier1_hortense.rst:645]: ok, nvm, using your suggestion

File: source/gent/tier1_hortense.rst (hunk 1 above)
Comment (wpoely86) [reply source/gent/tier1_hortense.rst:651]: We use `MaxNodes` for that purpose. 

You can also use multiple partitions for a job and let the users pick: either `gpu_rome_a100_40` or `gpu_rome_a100_80` or you specify both (or use this as default for when the partition has not been set for a GPU job). That's what we currently do on Hydra with a job submit plugin.

File: source/gent/tier1_hortense.rst [hunk 10]
```diff
@@ -12,19 +12,21 @@ General information
 **Hortense** is the 3rd VSC Tier-1 cluster, following *muk* (hosted by HPC-UGent, 2012-2016)
 and *BrENIAC* (hosted by HPC-Leuven, 2016-2022).
 
-The first phase of Hortense is also named ``dodrio``.
+The Hortense cluster also has an internal name ``dodrio``.
 
 It is available since 2021, is hosted by Ghent University,
 and maintained and supported by the HPC-UGent team.
 
+In 2023 a second phase was added, more than doubling the existing capacity of the system.
```
Comment (boegel) [inline source/gent/tier1_hortense.rst:20]: In May 2023?

File: source/gent/tier1_hortense.rst [hunk 11]
```diff
@@ -12,19 +12,21 @@ General information
...
 ----------------
 
-Hortense currently consists of 3+1 partitions:
+Hortense consists of the following partitions:
 
-- ``dodrio/cpu_rome``: main partition:
-   - 294 workernodes, each with:
+- ``dodrio/cpu_rome``: phase 1 main partition:
+   - 342 workernodes, each with:
        - 2x 64-core AMD Epyc 7H12 CPU 2.6 GHz (128 cores per node)
```
Comment (boegel) [inline source/gent/tier1_hortense.rst:30]:
```suggestion
       - 2x 64-core AMD Epyc 7H12 CPU 2.6 GHz ("Rome" microarchitecture, 128 cores per node)

File: source/gent/tier1_hortense.rst [hunk 12]
```diff
@@ -33,20 +35,32 @@ Hortense currently consists of 3+1 partitions:
...
        - 512 GiB RAM (~4GB/core), no swap
        - 480 GB SSD local disk
-- ``dodrio/gpu_rome_a100``: GPU partition:
+- ``dodrio/cpu_milan``: phase 2 main partition:
+   - 384 workernodes, each with:
+       - 2x 64-core AMD Epyc 7763 CPU 2.45 GHz (128 cores per node)
```
Comment (boegel) [inline source/gent/tier1_hortense.rst:40]: - 2x 64-core AMD Epyc 7763 CPU 2.45 GHz ("Milan" microarchitecture, 128 cores per node)

File: source/gent/tier1_hortense.rst [hunk 13]
```diff
@@ -526,7 +543,7 @@ The ``install.sh`` script should be implemented such that it installs the softwa
 with ``/readonly/``.
 
 Or you can start a new shell session in which ``/readonly/$VSC_SCRATCH_PROJECTS_BASE/...`` is
-accessible with write permissions:
+accessible with sort-of write permissions:
```
Comment (boegel) [inline source/gent/tier1_hortense.rst:546]: I would only add "sort-of" if you explain why you call it "sort-of"... That's already clarified above, no need to here imho.

File: source/gent/tier1_hortense.rst [hunk 14]
```diff
@@ -617,3 +634,37 @@ Getting help
 
 For questions and problems related to Tier-1 Hortense, please contact the central
 support address for Tier-1 compute: `compute@vscentrum.be <mailto:compute@vscentrum.be>`_.
+
+
+Phase 2
```
Comment (boegel) [inline source/gent/tier1_hortense.rst:639]:
```suggestion
Phase 2 (May 2023)
```

Maybe this should be under a separate "History" section, where you also mention something on `Phase 1`?

File: source/gent/tier1_hortense.rst [hunk 15]
```diff
@@ -617,3 +634,37 @@ Getting help
 
 For questions and problems related to Tier-1 Hortense, please contact the central
 support address for Tier-1 compute: `compute@vscentrum.be <mailto:compute@vscentrum.be>`_.
```
Comment (boegel) [inline source/gent/tier1_hortense.rst:636]: `Resources` and `Getting help` parts should remain as last sections?

File: source/gent/tier1_hortense.rst [hunk 16]
```diff
@@ -617,3 +634,37 @@ Getting help
...
+
+
+Phase 2
+-------
+
+In May 2023 a second phase was installed, adding 48 more nodes to the ``cpu_rome`` partition,
+20 extra GPU nodes with double the CPU and GPU memory in the new ``gpu_rome_a100_80`` partition and
```
Comment (boegel) [inline source/gent/tier1_hortense.rst:643]:
```suggestion
20 extra GPU nodes with double the CPU and GPU memory in the new ``gpu_rome_a100_80`` partition, and

File: source/gent/tier1_hortense.rst [hunk 17]
```diff
@@ -617,3 +634,37 @@ Getting help
...
+still use the same ``gpu_rome_a100`` partition that now includes all GPU nodes (and only select the
+``gpu_rome_a100_40`` or ``gpu_rome_a100_80`` for specific cases, e.g. when running multinode GPU jobs
+or explicitly requiring the larger amount of GPU/CPU memory of the ``gpu_rome_a100_80`` nodes).
+
+In the startup period, users are encouraged to try out the ``cpu_milan`` partition to compare performance
+and overal functioning with the ``cpu_rome`` partitions.
+In a later stage, projects will be given access to either the ``cpu_rome`` partitions
+or the ``cpu_milan`` partition.
+
+With both phases active, the cluster crossed the symbolic threshold of 100000 cores.
```
Comment (boegel) [inline source/gent/tier1_hortense.rst:659]: `100000` -> `100,000`

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by boegel** [comment]
@stdweird This needs more love before we can merge it?

File: source/gent/tier1_hortense.rst (hunk 10 above)
Comment (stdweird) [reply source/gent/tier1_hortense.rst:20]: previous paragraph is also vague, so i keep it the same

**Comment by epauwels** [comment]
This documentation looks great to me tbh
As far as I'm concerned no further changes needed

File: source/gent/tier1_hortense.rst (hunk 10 above)
Comment (epauwels) [reply source/gent/tier1_hortense.rst:20]: May is mentioned later on. This suffices imho

**Review by wdpypere — APPROVED** [review]

**Pull request merged**